src/platforms/esp32/components/avm_sys/include/otp_socket_platform.h

```diff
@@ -29,6 +29,10 @@ extern "C" {
 
 #define TAG "otp_socket"
 
+#define AVM_LOGD(tag, format, ...) \
+    ;
+// fprintf(stderr, "D %s: " format " (%s:%i)\n", tag, ##__VA_ARGS__, __FILE__, __LINE__);
```

Comment:
Is this a leftover?

Reply:
Sort of.  Since there is no way to configure log levels, I figured I'd leave these in place so they could be uncommented.  But since they are offensive, I have removed them.